Android/README.md

```diff
@@ -0,0 +1,46 @@
+# iTwin Mobile SDK Android Samples
+
+## Consistency Across Modules
+
+These samples make use of the iTwin [`mobile-sdk-android`](https://github.com/iTwin/mobile-sdk-android) package (which itself uses the iTwin [`mobile-native-android`](https://github.com/iTwin/mobile-native-android/releases) package), as well as the `@itwin/mobile-sdk-core` and `@itwin/mobile-ui-react` npm modules. In order to work properly, all of these things have to be in sync with each other. When new functionality is added to any of the dependent modules, and that new functionality is used in the samples, the samples won't work until the next official release of the dependent modules. That means that there is a very good chance that the main branch __will not work__ at any given time. To avoid problems when using these samples, you should always check out a release tag.
+
+## Client ID Setup
+
+Before building the samples, you must configure a Client ID for yourself. To do so, follow the instructions [here](../cross-platform/ClientID.md).
+
+## ITMSamples.properties Setup
+
+1. In the `Android/Shared` directory, copy the file `example-ITMSamples` and rename it to `ITMSamples.properties`. You will need to configure `itmapplication.client_id` and `itmapplication.scope`.
+
+    Notes:
+
+    * Do not surround your values with quotes.
+
+    * If you already had a Client ID, browse to your app's page on developer.bently.com. If you just created it, you should already be there.
```

Comment:
Make this a link (should be done in iOS/README.md too if you want)

Reply:
Make what a link?

Reply:
developer.bently.com

Reply:
That's not a useful link, and making it a link would (IMO) be bad. The useful link would be https://developer.bentley.com/my-apps/, but there's no nice way to include it here. The "your app's page" text refers to a link that would be visible from the my-apps list. But we can't link directly, because it is different for everybody. I'm open to suggestions.

Reply:
The my-apps link shows all the apps including their client id's so I think that would be sufficient.

Reply:
But how would that be worded? It doesn't make sense with the current wording.

Reply:
Maybe something like this?

If you already had a Client ID, browse to your app's page on [my-apps](https://developer.bently.com/my-apps) and copy it. If you just created it, you should already be there.

Reply:
Fixed.